### DISCUSSION TITLE
Smart System Statistics

### DISCUSSION THREAD

**Discussion by WolfoTheWolf** [pelican-dev/panel#39] [Ideas]
- Ability to view node performance, storage usage, CPU usage, memory usage, and possibly live logs of Wings or general panel log files. These log files do not need to be live but maybe a button that we can push and auto download them. For server statistics this should be able to be filtered down by game server and by node. Maybe able to see logins, invalid logins and such just to be able to monitor everything in one spot.

**Comment by WolfoTheWolf** [comment]
Also would like to pull wings or ptero logs within this page, and who knows, maybe even be able to filter by error or warning type

**Comment by notAreYouScared** [comment]
Are you looking for somthing like this?

```json
{
  "memory_total": 8323084288,
  "memory_used": 3584659456,
  "swap_total": 4294963200,
  "swap_used": 1323008,
  "load_average1": 0.25,
  "load_average5": 0.25,
  "load_average15": 0.45,
  "cpu_percent": 1.921378091877029,
  "disk_total": 41428451328,
  "disk_used": 18889232384
}
```

https://github.com/pelican-dev/wings/commit/12ac06d063094e2cbdb3c10550a8c8fc1c601b92

**Reply by WolfoTheWolf** [reply]
Yeah that would work

**Reply by notAreYouScared** [reply]
Wings already outputs this information, im sure we can show it on the node page. We already show CPU/Ram usage with charts that update every 5 seconds.

**Comment by notAreYouScared** [comment]
Live Logs might be a thing we can do with wings, and the panel. Will have to look into it.
As for the usages, They will stay as they're currently implemented, they might get refined over time, if you need more monitoring there are other tools built for that,

As for login attemptes you can see login attempts on your own account under the activity tab on your profile.

**Comment by delta-bee** [comment]
I'd also like to see this implemented into Pelican panel. I'd rather not use 3rd party tools, this doesn't seem that hard to implement.
